app/views/entity/list.html.php

```diff
@@ -1,7 +1,11 @@
 <h2>Lijst van Interesse's</h2>
 
 <ul>
-<?php foreach ($list as $item) : ?>
-   <li><?php echo $item->name ?></li>
-<?php endforeach ?>
+    <?php if (count($list) == 0) : ?>
```

Comment:
-Zou het hier niet mooier?beter(?) zijn om een if-else ervan te maken. if(count == 0) {} else (foreach loop). 
- moet count ==0 niet count===0 worden?
- is het mogelijk om ipv count, te check of the array empty met empty($list)? Is het een beter dan het ander?